### PR TITLE
Read and write PLY point clouds

### PR DESCRIPTION
**Pull request by sgiraudot**
Small feature: https://cgal.geometryfactory.com/CGAL/Members/wiki/Features/Small_Features/Point_set_processing_ply_io

### CODE REVIEW

File: Point_set_processing_3/include/CGAL/IO/read_ply_points.h [hunk 1]
```diff
@@ -0,0 +1,602 @@
...
+#include <CGAL/value_type_traits.h>
+#include <CGAL/point_set_processing_assertions.h>
+
+#include <boost/version.hpp>
+#if BOOST_VERSION >= 104000
+#include <boost/property_map/property_map.hpp>
+#else
+#include <boost/property_map.hpp>
+#endif
+
```
Comment (sloriot) [inline Point_set_processing_3/include/CGAL/IO/read_ply_points.h]: `#include <CGAL/property_map.h>` does this trick

File: Point_set_processing_3/include/CGAL/IO/read_ply_points.h [hunk 2]
```diff
@@ -0,0 +1,602 @@
+#ifndef CGAL_READ_PLY_POINTS_H
```
Comment (sloriot) [inline Point_set_processing_3/include/CGAL/IO/read_ply_points.h]: license header  is missing

File: Point_set_processing_3/include/CGAL/IO/write_ply_points.h [hunk 3]
```diff
@@ -0,0 +1,246 @@
+#ifndef CGAL_WRITE_PLY_POINTS_H
```
Comment (sloriot) [inline Point_set_processing_3/include/CGAL/IO/write_ply_points.h]: License header is missing

**Comment by sloriot** [comment]
Merged in CGAL-4.8-Ic-16

**Comment by sloriot** [comment]
Merged in [CGAL-4.8-Ic-17](https://cgal.geometryfactory.com/CGAL/Members/testsuite/results-4.8-Ic-17.shtml)

**Comment by sloriot** [comment]
Testsuite is missing the windows platforms

**Comment by lrineau** [comment]
Will be tested in [CGAL-4.8-Ic-19][t].

[t]: https://cgal.geometryfactory.com/CGAL/Members/testsuite/results-4.8-Ic-19.shtml

**Comment by sgiraudot** [comment]
Everything went fine on the testsuite. The only errors in _Point_set_processing_3_ come from _vcm_all_tests_ which is not affected by this branch. The _read_test_ file was compiled and executed with no issue.

**Comment by lrineau** [comment]
Will be tested in [CGAL-4.8-Ic-19][t].

[t]: https://cgal.geometryfactory.com/CGAL/Members/testsuite/results-4.8-Ic-19.shtml

**Comment by lrineau** [comment]
CGAL-4.8-Ic-19 was not a good release. And anyway there is no hurry because the small feature is not yet approved (only pre-approved for one week).

Will be tested in [CGAL-4.8-Ic-21][t].

[t]: https://cgal.geometryfactory.com/CGAL/Members/testsuite/results-4.8-Ic-21.shtml

**Comment by lrineau** [comment]
Merged in [CGAL-4.8-Ic-22](https://cgal.geometryfactory.com/CGAL/Members/testsuite/results-4.8-Ic-22.shtml).

**Comment by lrineau** [comment]
Merged in [CGAL-4.8-Ic-23](https://cgal.geometryfactory.com/CGAL/Members/testsuite/results-4.8-Ic-23.shtml).

**Comment by lrineau** [comment]
Now that `Three` is merged in `master`, you should cherry-pick the commit e510d701819c19a5139b8d3c8195ee98a91f9e74 in your branch.

**Comment by lrineau** [comment]
This pull-request was merged, but the commit CGAL/cgal-dev@e510d701819c19a5139b8d3c8195ee98a91f9e74 is still not cherry-picked. The plugin `ply_to_xyz` from `master` cannot compile.

**Comment by sloriot** [comment]
Done

**Pull request merged**